src/ast.rs

```diff
@@ -7450,6 +7458,26 @@ impl Parse for SpecialName {
 
                 Ok((SpecialName::JavaResource(resource_names), tail))
             }
+            b"GT" => {
+                match tail.peek() {
```

Comment:
Can you do `tail.next_or(error::Error::UnexpectedEnd)?`

Reply:
Done, that looks a lot better. Thanks for the suggestion.